javadoc task fails

After bumping Gradle to 7+, the task no longer works.

```
* What went wrong:
Execution failed for task ':javadoc'.
> class java.lang.String cannot be cast to class java.util.List (java.lang.String and java.util.List are in module java.base of loader 'bootstrap')


Caused by: java.lang.ClassCastException: class java.lang.String cannot be cast to class java.util.List (java.lang.String and java.util.List are in module java.base of loader 'bootstrap')
        at org.gradle.external.javadoc.CoreJavadocOptions.modulePath(CoreJavadocOptions.java:325)
        at org.gradle.api.tasks.javadoc.Javadoc.generate(Javadoc.java:147)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:58)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:51)
```

Error is caused at line https://github.com/gluonhq/attach/blob/master/build.gradle#L61.

If that line is commented out and we replace `-classpath` with `--module-path` in the `javadoc.options` file, running `javadoc @attach/build/tmp/javadoc/javadoc.options` works fine.

--- a/build.gradle
+++ b/build.gradle
@@ -58,7 +58,7 @@ task copyModuleSources {
 javadoc {
     doFirst {
         subprojects.each {p -> classpath += p.sourceSets.main.compileClasspath }
-        options.addStringOption('-module-path', classpath.asPath)
+        options.modulePath = new ArrayList(classpath.files)
     }
     exclude("com/gluonhq/attach/*/impl")
     source fileTree("${buildDir}/modules/sources")

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

